ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.12-periodics.yaml

```diff
@@ -273,3 +273,95 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 11 2 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: v1.12
+    org: redhat-developer
+    repo: gitops-operator
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: gitops-ocp4.17-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-developer-gitops-operator-v1.12-gitops-ocp4.17-lp-interop-redhat-openshift-gitops-interop-aws
```

Comment:
```suggestion
  name: periodic-ci-redhat-developer-gitops-operator-v1.13-gitops-ocp4.17-lp-interop-redhat-openshift-gitops-interop-aws
```

Reply:
@varshab1210 - We ported these changes (for GitOps 1.13 on OCP 4.17 only) to [PR#54131](https://github.com/openshift/release/pull/54131).